ldms/man/ldmsd_sampler_advertisement.man

```diff
@@ -47,7 +47,7 @@ ldmsd_sampler_advertisement - Manual for LDMSD Sampler Advertisement
 
 .SH DESCRIPTION
 
-LDMSD Sampler Discovery is a capability that enables LDMSD automatically add
+LDMSD Sampler Advertisement is a capability that enables LDMSD automatically add
 producers that its hostname matches a given regular expression. The feature
 eliminates the need for manual configuration of sampler hostname in the
```

Comment:
reduces the need for the manual configuration of producers in configuration files